src/KDSpdSetup/details.h

```diff
@@ -312,6 +315,10 @@ class SPDMaps
 #define createMsvcSinkMtPtr createMsvcSinkPtr<std::mutex>
 #endif
 
+#define createRingBufferSinkStPtr createRingBufferSinkPtr<spdlog::details::null_mutex>
```

Comment:
Could we use a "using" instead of a #define for these? Or does that make something down the line more tricky?